docs/_posts/examples/3400-01-31-data-join.html

```diff
@@ -0,0 +1,70 @@
+--- 
+layout: example 
+category: example 
+title: Join client data to vector tiles 
+description: Use data-driven styles to join client data to vector tiles
+tags: 
+  - sources 
+---
+
+<div id='map'></div>
+<script>
+var map = new mapboxgl.Map({
+    container: 'map',
+    style: 'mapbox://styles/mapbox/light-v9',
+    center: [-99.9, 41.5],
+    zoom: 1
+});
+
+// Generate client data to join to vector tiles
+// Key is the property to join, value is the measure to style on
+// EX: {"01" : 27, "02" : 89}
```

Comment:
Do you have any real data sets we could use with this example? 

It may improve the readability of the example to have more concrete terminology (ie. "Calculate color based on the GDP" vs "Calculate color based on the data value") and remove the data generation step.